Btw, the `build-test-mudblazor / format-check (pull_request)` is failing with:

```
/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/Components/DatePicker/MudDatePicker.cs(38,1): error WHITESPACE: Fix whitespace formatting. Replace 17 characters with '\n\s\s\s\s\s\s\s\s'. [/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/MudBlazor.csproj]
/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/Components/DatePicker/MudDatePicker.cs(163,22): error WHITESPACE: Fix whitespace formatting. Replace 21 characters with '\n\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s'. [/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/MudBlazor.csproj]
/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/Components/DatePicker/MudDatePicker.cs(164,147): error WHITESPACE: Fix whitespace formatting. Replace 21 characters with '\n\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s'. [/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/MudBlazor.csproj]
/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/Components/DatePicker/MudDatePicker.cs(174,50): error WHITESPACE: Fix whitespace formatting. Replace 21 characters with '\n\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s'. [/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/MudBlazor.csproj]
/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/Components/DatePicker/MudDatePicker.cs(175,124): error WHITESPACE: Fix whitespace formatting. Replace 21 characters with '\n\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s'. [/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/MudBlazor.csproj]
/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/Components/DatePicker/MudDatePicker.cs(203,22): error WHITESPACE: Fix whitespace formatting. Replace 21 characters with '\n\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s'. [/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/MudBlazor.csproj]
/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/Components/DatePicker/MudDatePicker.cs(204,147): error WHITESPACE: Fix whitespace formatting. Replace 21 characters with '\n\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s'. [/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/MudBlazor.csproj]
/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/Components/DatePicker/MudDatePicker.cs(214,50): error WHITESPACE: Fix whitespace formatting. Replace 21 characters with '\n\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s'. [/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/MudBlazor.csproj]
/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/Components/DatePicker/MudDatePicker.cs(215,81): error WHITESPACE: Fix whitespace formatting. Replace 21 characters with '\n\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s'. [/home/runner/work/MudBlazor/MudBlazor/src/MudBlazor/MudBlazor.csproj]
Error: Process completed with exit code 2.
```

--- a/src/MudBlazor/Components/DatePicker/MudDatePicker.cs
+++ b/src/MudBlazor/Components/DatePicker/MudDatePicker.cs
@@ -35,7 +35,7 @@ namespace MudBlazor
 
         protected Task SetDateAsync(DateTime? date, bool updateValue)
             => SetDateAsync(date, updateValue, false);
-        
+
         protected async Task SetDateAsync(DateTime? date, bool updateValue, bool forceUpdate)
         {
             if (_value != null && date != null && date.Value.Kind == DateTimeKind.Unspecified)
@@ -48,7 +48,7 @@ namespace MudBlazor
             /* See #7866 for more details
              * When the date is set in the UI, this method gets called with the same value multiple time. This guard
              * debounces the value to the same value in a short time frame is ignored. The debounce is ignored if
-             * foreceUpdate is true
+             * forceUpdate is true
              */
             if (_value == date && (now - _lastSetTime).TotalMilliseconds < DebounceTimeoutMs && !forceUpdate)
             {

